egs/librispeech/ASR/pruned_transducer_stateless7/finetune.py

```diff
@@ -104,6 +103,7 @@ def set_batch_count(model: Union[nn.Module, DDP], batch_count: float) -> None:
 
 def add_finetune_arguments(parser: argparse.ArgumentParser):
     parser.add_argument("--do-finetune", type=str2bool, default=False)
+    parser.add_argument("--use-mux", type=str2bool, default=False)
```

Comment:
Could you please add some documentation for this argument?

Reply:
Sure